addons/website/static/src/snippets/s_mega_menu_menu_image_menu/000.scss

```diff
@@ -0,0 +1,11 @@
+
+.s_mega_menu_menu_image_menu:not([data-vcss]) {
+    // Adjust the layout for the vertical header
+    @if o-website-value('header-template') == 'sidebar' {
+        @include media-breakpoint-up(lg)  {
+            > .container > .row {
+                flex-direction: column !important;
+            }
+        }
+    }
```

Comment:
Hi @qsm-odoo , I agree with you that no user, safe in his mind, should ever use our mega menu with a side bar.
I don't want you to either spend energy to prevent it or either spend energy to improve it.
Let's just leave it as-is.

Reply:
Ok I'll simplify the code then :+1: